### ISSUE TITLE
Example of a second-level analysis on scalar (e.g. T1) images

### ISSUE THREAD

**Issue by demianw** [nistats/nistats#191]
Hey! I think it would be informative to have a simple example performing a second-level analysis on scalar images not issued from fMRI. It will provide potential users with an easier-access tutorial for more general analyses which are not uncommon in other sub-fields of neuroimaging.

Do you think it's worth it to have one like this?

**Comment by bthirion** [comment]
Yes. I'd suggest to start with OASIS, unless you have a better suggestion.

**Comment by demianw** [comment]
Sounds good with me. How do we go about implementing this?

**Comment by bthirion** [comment]
We write a script that does the job and then we improve it. Do you have time this week or next  Monday/Tuesday to work on it ?

**Comment by demianw** [comment]
Oops missed this one. Next monday/tuesday should work for me.

**Comment by bthirion** [comment]
Sorry, I'm in vacation till Aug. 13th --with bad internet connection. Would Aug. 13th work for you ?

**Comment by demianw** [comment]
Yup!

D
--
Demian Wassermann, PhD
demian.wassermann@inria.fr
Associate Research Professor (CRN)
Parietal Team
INRIA Saclay Ile-de-France
1 Rue Honoré d'Estienne d'Orves, 91120 Palaiseau
